### PR TITLE
Run scalability envelope everyday instead of once a week

### PR DESCRIPTION
**Pull request by rkooo567**
Also move scalability envelope to daily tests. cc @wuisawesome

### CODE REVIEW

**Review by wuisawesome — APPROVED** [review]
@krfricke I'll let you merge this one, since I think it requires some additional setup on the buildkite side.

**Comment by rkooo567** [comment]
@krfricke This PR's purpose was changed. Will merge it after resolving merge conflict.

**Pull request merged**
